dotcom-rendering/src/web/components/Callout/CalloutComponents.tsx

```diff
@@ -144,15 +174,19 @@ export const CalloutShare = ({ format }: Props) => {
 const termsAndConditionsStyles = (format: ArticleFormat) =>
 	css`
 		a {
-			color: ${decidePalette(format).text.calloutAccent};
-			border-bottom: 1px solid ${decidePalette(format).text.calloutAccent};
+			color: ${decidePalette(format).text.richLink};
```

Comment:
I noticed the palette methods `textCallout` and `textRichLink` used in this PR are identical. Do we need two methods? I know there's a lot of duplication in the palette anyway, just raising the question here in case this is unintentional?

Reply:
It is intentional - but i'm open to going with whatever you think is the best way? Because there is a lot of duplication in the file, I kind of assumed it was the standard to duplicate whenever you intend to use `decidePalette` in a new place